lms/static/js/edxnotes/views/tab_view.js

```diff
@@ -122,10 +122,14 @@ define([
          * Shows error message.
          */
         showErrorMessage: function (message) {
-            this.$('.wrapper-msg')
-                .removeClass('is-hidden')
-                .find('.msg-content .copy').html(message);
+            var $wrapper = this.$('.wrapper-msg');
+            $wrapper.removeClass('is-hidden');
+
+            if (!(message instanceof HtmlUtils.HtmlSnippet)) {
```

Comment:
I don't agree on the dangerous part. I think it makes it impossible to set raw html that is not escaped. However I agree that caller should do it in first place for template linter. If it wasn't for linter I would prefer to use this form as it just ensures the safe html no matter if it's called with raw html string or escaped one.